variables.tf

```diff
@@ -68,6 +68,13 @@ variable "repo_lifecycle_policy" {
   nullable    = true
 }
 
+variable "repo_policy" {
```

Comment:
@a1k0u, please run `terraform-docs .`, so that the `README.md` is updated with the new variable (and other variable description is updated)

Reply:
eh.. not sure how I missed that it was already executed.. I think I'm done for today